### PR TITLE
feat: tag feature on creation

### PR DESCRIPTION
**Pull request by sjaanus**
Now it is possible to tag feature on creation.

### CODE REVIEW

File: src/lib/features/feature-toggle/feature-toggle-service.ts [hunk 1]
```diff
@@ -1256,7 +1256,7 @@ class FeatureToggleService {
         await this.validateName(value.name);
         await this.validateFeatureFlagNameAgainstPattern(value.name, projectId);
 
-        const exists = await this.projectStore.hasProject(projectId);
+        const projectExists = await this.projectStore.hasProject(projectId);
```
Comment (sjaanus) [inline src/lib/features/feature-toggle/feature-toggle-service.ts:1259]: Make this more clear

File: src/lib/features/feature-toggle/feature-toggle-service.ts [hunk 2]
```diff
@@ -1301,6 +1301,16 @@ class FeatureToggleService {
                 );
             }
 
+            if (value.tags && value.tags.length > 0) {
```
Comment (sjaanus) [inline src/lib/features/feature-toggle/feature-toggle-service.ts:1304]: If exist in payload, insert tags.

File: src/lib/openapi/spec/create-feature-schema.ts [hunk 3]
```diff
@@ -32,14 +33,17 @@ export const createFeatureSchema = {
         },
         tags: {
             type: 'array',
+            description: 'Tags to add to the feature.',
             items: {
-                type: 'string',
-                example: 'simple:test',
+                $ref: '#/components/schemas/tagSchema',
```
Comment (sjaanus) [inline src/lib/openapi/spec/create-feature-schema.ts:38]: Fixing the schema to reuse existing schema and make it similar across unleash.

Comment (thomasheartman) [reply src/lib/openapi/spec/create-feature-schema.ts:38]: Oh, wow, did we actually have the possibility to do this before?

**Review by thomasheartman — APPROVED** [review]

File: src/lib/openapi/spec/create-feature-schema.ts (hunk 3 above)
Comment (thomasheartman) [reply src/lib/openapi/spec/create-feature-schema.ts:38]: Wait, does this break the API somehow? As in, is this a breaking change?

Comment (sjaanus) [reply src/lib/openapi/spec/create-feature-schema.ts:38]: Nope, I implemented it yesterday, but after adding backend code, it made sense to use tagSchema.

**Pull request merged**